samples/react-application-nav-search-driven/package.json

```diff
@@ -13,29 +13,29 @@
   },
   "dependencies": {
     "@fluentui/react": "8.106.4",
-    "@microsoft/decorators": "1.18.2",
+    "@microsoft/decorators": "1.20.0",
     "@microsoft/generator-sharepoint": "^1.19.0",
-    "@microsoft/sp-adaptive-card-extension-base": "1.18.2",
-    "@microsoft/sp-application-base": "1.18.2",
-    "@microsoft/sp-core-library": "1.18.2",
-    "@microsoft/sp-dialog": "1.18.2",
+    "@microsoft/sp-adaptive-card-extension-base": "1.20.0",
+    "@microsoft/sp-application-base": "1.20.0",
+    "@microsoft/sp-core-library": "1.20.0",
+    "@microsoft/sp-dialog": "1.20.0",
     "@types/react": "17.0.45",
     "@types/react-dom": "17.0.17",
```

Comment:
this was a mistake. all types should be a devDependency. Lets move it as well

Reply:
I have update the devDependency